ofrak_core/ofrak/gui/server.py

```diff
@@ -410,39 +457,66 @@ async def queue_patch(self, request: Request) -> Response:
         start = int(start_param) if start_param is not None else 0
         end_param = request.query.get("end")
         end = int(end_param) if end_param is not None else (await resource.get_data_length())
-
+        # TODO: There has to be a better way
+        new_data_string = "/x" + "/x".join(
```

Comment:
Maybe I'm misunderstanding, but should this be a backslash?

```suggestion
        new_data_string = "\\x" + "\\x".join(
```

Reply:
yes...